recipes/tributary/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "tributary" %}
+{% set version = "0.1.0rc2" %}
```

Comment:
Please only PR only full releases of a package.  RCs have a different process. Check out the CFEP for that

Reply:
bumped to 0.1.0